apps/sanity/schemas/schedule.ts

```diff
@@ -0,0 +1,39 @@
+import { defineType, defineField} from "sanity";
+
+export default defineType({
+	name: "schedule",
```

Comment:
Should this schema be named something like `event` instead?

Reply:
fixed!